runtime/src/cost_tracker.rs

```diff
@@ -35,6 +39,10 @@ pub struct CostTracker {
     vote_cost: u64,
     transaction_count: u64,
     account_data_size: u64,
+
+    /// The amount of total account data size remaining.  If `Some`, then do not add transactions
+    /// that would cause `account_data_size` to exceed this limit.
+    account_data_size_limit: Option<u64>,
```

Comment:
So this limit is optional because it depends on a feature? If so, to be super neat, may as well change all the other limits to `Option<>`, and update calls that passing `MAX` with `None`. But can leave it to later time